README.md

```diff
@@ -102,7 +102,7 @@ For convenience, the CredHub team runs a public UAA whose IP is in the default `
 #### Running CredHub with local UAA
 
 In order to run CredHub against a UAA running on your local machine, do the following:
-1. Start a UAA with Docker: `docker run -d --mount type=bind,source=$PWD/config/uaa.yml,target=/uaa/uaa.yml -p 127.0.0.1:8080:8080 pcfseceng/uaa:latest`
+1. Start a UAA with Docker: `docker run -d -v $(PWD)/config/uaa.yml:/uaa/uaa.yml -p 127.0.0.1:8080:8080 pcfseceng/uaa:latest`. Also may need to add the config/uaa.yml path to the Docker Settings-Resources-FileSharing permissions. 
```

Comment:
The existing command options work fine for me. What was the issue that required you to make this change?

Reply:
This was the error: 

docker: Error response from daemon: invalid mount config for type "bind": bind source path does not exist: ...config/uaa.yml.

I see now that the change to the docker command is not necessary, but on my system the path to the config directory must be added to Docker Settings-Resources-File sharing Virtual File shares.  Without it, the above error occurs.

Reply:
> but on my system the path to the config directory must be added to Docker Settings-Resources-File sharing Virtual File shares

I did not have to do this. Anyhow, I suggest to the keep the docker command options as it was but just add the additional clause about `config/uaa.yml path`.